src/main/java/com/github/krzysiekjodlowski/ox/Settings.java

```diff
@@ -0,0 +1,26 @@
+package com.github.krzysiekjodlowski.ox;
+
+/**
+ * Keep settings for whole game. As a singleton
+ * it is reachable from everywhere in the game.
+ * ATTENTION - This very truncated implementation
+ * is temporary and will change in the future.
+ *
+ * @author Krzysztof Jodlowski
+ */
+public enum Settings {
+  INSTANCE;
+
+  static final int MIN_GAME_BOARD_SIDE_LENGTH = 3;
+  static final int MAX_GAME_BOARD_SIDE_LENGTH = 40;
+  static final int FIRST_FIELD_NUMBER = 1;
+  static int gameBoardSideLength = 3;
```

Comment:
There is no need of storing this value in Settings. I mentioned it in OXConsoleGame class comment.
It would be dangerous if we want to handle multiple games at once because Settings is singleton and each game would be modifying value of this variable.

Reply:
Ok, I've removed boardSideLength and boardCapacity() from settings, but in cost of providing a public getter for capacity. I could count boardSideLength * boardSideLength gained from user tough.